CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 
 * ![Bugfix][badge-bugfix] Documenter now generates the correct source URLs for docstrings from other packages when the `repo` argument to `makedocs` is set (note: the source links to such docstrings only work if the external package is cloned from GitHub and added as a dev-dependency). However, this change **breaks** the case where the `repo` argument is used to override the main package/repository URL, assuming the repository is cloned from GitHub. ([#1808][github-1808])
 
+* ![Enhancement][badge-enhancement] Added Woodpecker CI as another CI solution for deploying docs. ([#1880][github-1880])
+
```

Comment:
I created some conflicts again by merging a few PRs. Could you also move this up the list a little, so that it would be the last of the enhancements?